### ISSUE TITLE
data example

### ISSUE THREAD

**Issue by wjlszhang** [ziyangyeh/Auto-Orth#2]
Dear

It's exciting since I'm following the TANet implmementation for serveral months, thanks a lot for sharing.
however, could you please share some data examples here? including model .stl and label .txt, if not convenience here,  woud you pls reach me wjlszhang@126.com. 

Much appreciated!

**Comment by smoothumut** [comment]
hi @ziyangyeh , 
thanks for the great works and sharings. Both TANet and iMeshSegNet are great.
For me, to make Tanet work and test I would love to get data example too if it is possible. 
thanks thanks in advance

**Comment by ziyangyeh** [comment]
@smoothumut 
I don't have any examples currently. I think you can ask other issuers for help.
It is easy to figure out how training data are generated from the dataset pipeline and the paper itself.

**Comment by smoothumut** [comment]
Hi @wjlszhang 
Do you have a sample of data_set.csv file with 1 or 2 files in it?
Thanks in advance

**Comment by smoothumut** [comment]
Hi @ziyangyeh 
I got it. By the way, thanks for the very clean and structured code. I have one quick question. In labels text file, lets say upper jaw labels, do we give labels for each face or each tooth in that txt file? So do we segment the teeth and input that data or do the preprocessing section segment the teeth? I couldnt understand that part : )
thanks in advance ziyangyeh, 
best regards
Umut

**Comment by wangduyang** [comment]
> 



> Hi @ziyangyeh I got it. By the way, thanks for the very clean and structured code. I have one quick question. In labels text file, lets say upper jaw labels, do we give labels for each face or each tooth in that txt file? So do we segment the teeth and input that data or do the preprocessing section segment the teeth? I couldnt understand that part : ) thanks in advance ziyangyeh, best regards Umut

Hello, do you know what the training  data example  format is? Can you send me a copy to learn? Thank you
thanks in advance 1720186093@qq.com
best regards

**Comment by Wangzhijie3** [comment]
> Hi @ziyangyeh I got it. By the way, thanks for the very clean and structured code. I have one quick question. In labels text file, lets say upper jaw labels, do we give labels for each face or each tooth in that txt file? So do we segment the teeth and input that data or do the preprocessing section segment the teeth? I couldnt understand that part : ) thanks in advance ziyangyeh, best regards Umut

Hello, do you know what the training data example format is? Can you send me a copy to learn? Thank you
thanks in advance 1055029796@qq.com
best regards

**Comment by 1315975895** [comment]
@Wangzhijie3 hello，zhijie，have you got the data example format？It would be excellent if you could share it with me.(1315975895@qq.com)thank you so much.
